data/json/recipes/armor/arms.json

```
@@ -136,7 +154,8 @@
     "book_learn": [ [ "recipe_melee", 2 ] ],
     "using": [ [ "cordage", 1 ] ],
     "qualities": [ { "id": "HAMMER", "level": 1 }, { "id": "SAW_M", "level": 1 } ],
-    "components": [ [ [ "scrap", 60 ] ] ]
+    "components": [ [ [ "scrap", 60 ] ] ],
+    "proficiencies": [ { "proficiency": "prof_armorsmithing", "required": false, "time_multiplier": 3 } ]
```

```suggestion
    "proficiencies": [ { "proficiency": "prof_armorsmithing", "required": false, "time_multiplier": 1.5 },
    "proficiencies": [ { "proficiency": "prof_articulation", "required": false, "time_multiplier": 1.5 } ]
```
For scrap armour I kept the requirements low because you're not making it from scratch, not doing blacksmithing, etc.